runtime/compiler/runtime/J9Profiler.hpp

```diff
@@ -633,20 +633,22 @@ class TR_BlockFrequencyInfo
    int32_t getFrequencyInfo(TR::Block *block, TR::Compilation *comp);
    int32_t getFrequencyInfo(TR_ByteCodeInfo &bci, TR::Compilation *comp, bool normalizeForCallers = true, bool trace = true);
    void    setCounterDerivationInfo(TR_BitVector **counterDerivationInfo) { _counterDerivationInfo = counterDerivationInfo; }
+   TR_BitVector ** getCounterDerivativeInfo() { return _counterDerivationInfo; }
    void    setEntryBlockNumber(int32_t number) { _entryBlockNumber = number; }
    bool    isJProfilingData() { return _counterDerivationInfo != NULL; }
    static int32_t *getEnableJProfilingRecompilation() { return &_enableJProfilingRecompilation; }
    static void    enableJProfilingRecompilation() { _enableJProfilingRecompilation = -1; }
-
+   TR::Node* generateBlockRawCountCalculationNode(TR::Compilation *comp, int32_t blockNumber, TR::Node *node);
```

Comment:
Probably better to say SubTree rather than Node since it is a tree of nodes, but isn't a treetop.

Reply:
Changed it : https://github.com/eclipse/openj9/pull/2348/commits/dcd4b9def6654084825294d884774145fe894290